cs-utilities/pom.xml

```diff
@@ -140,11 +140,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
```

Comment:
need to update the dependency instead of removing it from the project.

Reply:
added the dependency